modin/pandas/test/test_series.py

```diff
@@ -3384,22 +3244,16 @@ def test_subtract(data):
     test_data_values + test_data_small_values,
     ids=test_data_keys + test_data_small_keys,
 )
-@pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
```

Comment:
Why removed?

Reply:
It doesn't make sense for the series. I can write a separate test just for this case.

Reply:
Test added.

Reply:
Ah, I see. I think we don't even need a test for this😄